Rakefile

```diff
@@ -15,6 +15,10 @@ Rake::TestTask.new do |t|
 end
 task :default => [:compile, :test]
 
+Rake::Task[:clean].enhance do
+  FileUtils.rm_rf(File.read(".gitignore").lines.flat_map { |l| Dir["**/#{l.chomp}"] })
```

Comment:
This is a followup to https://github.com/arthurnn/memcached/pull/202. Otherwise my change would still not apply.